app/services/idv/steps/ssn_step.rb

```diff
@@ -11,7 +11,7 @@ def call
 
       def extra_view_variables
         {
-          applicant_ssn: flow_session[:pii_from_doc][:ssn],
+          updating_ssn: flow_session[:pii_from_doc].has_key?(:ssn),
```

Comment:
The intention is switching to a boolean because it's easier to understand the code. I don't believe it's possible for SSN to be nil so I was ok with the change in logic.

Reply:
Based on review of the code around this, I think this scenario could possibly be produced by re-submitting the SSN form after reaching the address step. I'll take a moment after lunch to try reproducing the issue.

Reply:
Yes, that looks good. I did test the use case I was thinking of, though, and found a different error than I expected - the `Back` button gave me an error `undefined method '[]' for nil:NilClass` pointing to [Idv::Actions::CancelUpdateSsnAction.call](https://github.com/18F/identity-idp/blob/sbachstein/lg-6349-add-verify-page/app/services/idv/actions/cancel_update_ssn_action.rb#L5).

Reply:
Yeah the back and cancel buttons are broken in the IPP flow. LG-6163 and LG-6377 are the ticket where we're planning to make them work correctly

Reply:
Thanks, I added the `.present?` approach in 35c05f913